src/makeCalculator.js

```diff
@@ -37,7 +37,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    add(num) {
+      calculator.result += num;
+
+      return calculator;
+    },
+
+    subtract(num) {
+      calculator.result -= num;
+
+      return calculator;
+    },
+
+    multiply(num) {
+      calculator.result *= num;
+
+      return calculator;
+    },
+
+    divide(num) {
+      calculator.result /= num;
```

Comment:
it'd be nice to add a condition that num shouldn't be a zero. In opposite case you may return an error